.github/workflows/ci.yaml

```
@@ -0,0 +1,20 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  pre-commit-checks:
+    name: Pre-commit Checks
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+      - name: Set up pixi
+        uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659
```

```suggestion
        uses: prefix-dev/setup-pixi@v0
```

this should save some bytes as well

---

thank you for your valuable feedback! fixed in https://github.com/cmdevries/ToolHelperUtils/pull/3/commits/0000000e3e98a924131bd6d0f44fc666b804335b